modules/product/src/test/java/com/opengamma/strata/product/rate/BRLCompoundedOvernightRateComputationTest.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (C) 2018 - present by OpenGamma Inc. and the OpenGamma group of companies
+ *
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.product.rate;
+
+import static com.opengamma.strata.basics.index.OvernightIndices.BRL_CDI;
+import static com.opengamma.strata.basics.index.OvernightIndices.USD_FED_FUND;
+import static com.opengamma.strata.collect.TestHelper.assertSerialization;
+import static com.opengamma.strata.collect.TestHelper.assertThrowsIllegalArg;
+import static com.opengamma.strata.collect.TestHelper.coverBeanEquals;
+import static com.opengamma.strata.collect.TestHelper.coverImmutableBean;
+import static com.opengamma.strata.collect.TestHelper.date;
+import static org.testng.Assert.assertEquals;
+
+import org.testng.annotations.Test;
+
+import com.google.common.collect.ImmutableSet;
+import com.opengamma.strata.basics.ReferenceData;
+import com.opengamma.strata.basics.index.Index;
+import com.opengamma.strata.basics.index.OvernightIndexObservation;
+
+/**
+ * Test.
+ */
+@Test
+public class BRLCompoundedOvernightRateComputationTest {
```

Comment:
Strata class names never have adjacent upper case letters. So if this remains it would be `Brl` not `BRL`. Same with method names.